rules.txt

```diff
@@ -982,6 +988,8 @@
 /foster-?grandparents                     301  /programs/foster-grandparents/
 /fr$                                      301  /fr/ 
 /free-?tax-?help                          301  /services/payments-assistance-taxes/payment-plans-and-assistance-programs/income-based-programs-for-residents/find-free-tax-preparation-assistance-for-qualifying-taxpayers/
+/frontdoor                                301  https://housing-front-door.phila.gov/
+/front-door                               301  https://housing-front-door.phila.gov/
```

Comment:
Here you can make this a one-line redirect by using the ? regex!

/front-?door works for both URLs